app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -82,6 +82,10 @@ def export
         fail HttpErrors::BadRequest, _("ISO export must be enabled when specifying ISO size")
       end
 
+      if (repos = @version.content_view.on_demand_repositories).any?
+        fail HttpErrors::BadRequest, _("This content view has on-demand repositories that cannot be exported: %{repos}" % {repos: repos.pluck(:label).join(", ")})
```

Comment:
nitpick: Since hammer/UI refer to this as `on demand`, can we remove the hyphen?